jira_access/_types.py

```diff
@@ -0,0 +1,4 @@
+from typing import Dict, List, Union
+
+JsonType = Union[None, int, float, str, bool, List["JsonType"], Dict[str, "JsonType"]]
+Json = Dict[str, JsonType]
```

Comment:
Can you explain the purpose for this?

JsonType seems to contain as well Dict[str, JsonType]  ? 
(Assuming that "JsonType" is evaluated into the JsonType at runtime)

Reply:
A complete Json object is always a mapping from a string to a JsonType. If some function received as an input a Json object and we would annotate it as a JsonType, the type checker would complain, as soon as we would index into the json "dict" as it could be either None, Int, etc. Using the type "Json" we constrain it to always be a "dict" and therefore be a valid Json object.